src/runtime_src/core/pcie/driver/linux/xocl/userpf/xocl_bo.c

```diff
@@ -917,14 +918,13 @@ int xocl_info_bo_ioctl(struct drm_device *dev,
 
 static int xocl_migrate_unmgd(struct xocl_dev *xdev, uint64_t data_ptr, uint64_t paddr, size_t size, bool dir)
 {
-	int channel;
-	struct drm_xocl_unmgd unmgd;
-	int ret;
-	ssize_t migrated;
+	int channel = 0;
+	struct drm_xocl_unmgd unmgd = {0};
```

Comment:
zero (0) not needed.

```suggestion
	struct drm_xocl_unmgd unmgd = {};
```